src/components/ProfileSidebar.vue

```diff
@@ -30,7 +30,7 @@ const { web3Account } = useWeb3();
         <ProfileSidebarHeaderSkeleton v-else />
 
         <div
-          v-if="userAddress === web3Account"
+          v-if="userAddress === web3Account.toLocaleLowerCase()"
```

Comment:
```suggestion
          v-if="userAddress === web3Account.toLowerCase()"
```
Why is `Locale` needed?

Reply:
Auto completion gone wrong